.github/workflows/pact-verify.yml

```diff
@@ -1,73 +0,0 @@
-# Pact verify workflow
```

Comment:
There's a reference to this file in `.github/workflows/ci.yml`.  You'll need to delete that too, otherwise CI won't run.